### ISSUE TITLE
.grconfig.json should be in the project folder

### ISSUE THREAD

**Issue by alexgorbatchev** [mixu/gr#8]
It's currently not possible to share the `.grconfig.json` between developers because it contains full paths. Ideally it should be local to the project folder.

**Comment by nichtich** [comment]
Yes and pathes should be local to the location of `.grconfig.json` and `gr` should walk up the directory try to find `.grconfig.json` with home directory as last choice:

    /my/dir/project/.grconfig.json
    /my/dir/.grconfig.json
    /my/.grconfig.json
    /.grconfig.json # maybe skip this as it would be very bad practice
    /home/user/.grconfig.json

**Comment by nichtich** [comment]
Together with #23 one could easily copy .grconfig.json to some location and checkout a bunch of repositories with one command.

**Comment by epcim** [comment]
+1 
alternatively the "gr" should only work with repos under current $PWD. Force the behaviour by flag "-g" etc..

re: placing it in project repo (ie in root of superrepo) is fine, except you are on path elsewhere (one of the subrepo).

**Comment by coderjoe** [comment]
I'm not against the idea but it would very much gr's usefulness to me.

**Comment by RX14** [comment]
I think it's possible to have both, and simply extend gr's functionality instead of change it completely.
